test-data/unit/check-unions.test

```diff
@@ -1324,7 +1324,8 @@ class C10: ...
 
 x: Union[C1, C2, C3, C4, C5, C6, C7, C8, C9, C10, int]
 y: Union[C1, C2, C3, C4, C5, C6, C7, C8, C9, C10, str]
-x = y  # E: Incompatible types in assignment (expression has type "Union[C1, C2, C3, C4, C5, C6, C7, C8, C9, C10, str]", variable has type "Union[C1, C2, C3, C4, C5, C6, C7, C8, C9, C10, int]")
+x = y  # E: Incompatible types in assignment (expression has type "Union[C1, C2, C3, C4, C5, C6, C7, C8, C9, C10, str]", variable has type "Union[C1, C2, C3, C4, C5, C6, C7, C8, C9, C10, int]") \
+       # N: Subtype items that may cause the mismatch: "str"
```

Comment:
This is great, and allays my concern. Thanks! Though ideally I think it would be best if we could say something like "items in the first union not in the second" rather than "subtype items", which might not be familiar terminology to all of our users.

Reply:
Yeah, I was thinking about this, but then I thought that it is not technically correct, e.g. `Sub| int` is still a subtype of `Super | int`, but now it seems to me I am overthinking it. I will change the message.